variables.tf

```diff
@@ -106,6 +106,11 @@ variable "ecs_service_task_port" {
   default     = 3000
 }
 
+variable "ecs_service_worker_task_port" {
+  description = "The port the worker is listening on (ALB will map these with ephemeral port numbers)"
+  default     = 3000
```

Comment:
Is the default sidekiq port 4567, could this be conflicted with puma on 3000?

Reply:
The actual port is `8082`, this was my initial configuration but after going through the sidekiq code I tweaked it via the edge tfvars but missed adding the entry to the `workspace.tfcars.example`

`ecs_service_worker_task_port = 8082`